### ISSUE TITLE
DimensionByValue throws System.ArgumentException: Column out of range

### ISSUE THREAD

**Issue by kemsky** [EPPlusSoftware/EPPlus#1728]
### EPPlus usage

Commercial use (I have a commercial license)

### Environment

windows, ubuntu

### Epplus version

7.5.1

### Spreadsheet application

openoffice, excel

### Description

We upgraded 7.4.1 =\> 7.5.1. The old version did return expected number of rows in `Worksheet.Dimension`. New version returns 999999 rows. We checked excel file and indeed there are blank cells like this:

```xml
<row r="999962" spans="2:2" x14ac:dyDescent="0.3"><c r="B999962"/></row>
``` 

So at first we tried to filter manually, then discovered `DimensionByValue` which seems to be the thing that we needed but it did not work (col == 0):

```csharp
System.ArgumentException: Column out of range
   at OfficeOpenXml.ExcelRange.ValidateRowCol(Int32 Row, Int32 Col)
   at OfficeOpenXml.ExcelRange.get_Item(Int32 Row, Int32 Col)
   at OfficeOpenXml.ExcelWorksheet.get_LastValueCell()
   at OfficeOpenXml.ExcelWorksheet.get_DimensionByValue()
   at Common.Files.Parsing.Excel.ExcelParser`2.ParseWorkSheet(ExcelContext`1 context, List`1 result, List`1 errors, List`1 headers) 
```

**Comment by swmal** [comment]
Can you please provide a workbook or code that replicates this error?

**Comment by kemsky** [comment]
@swmal, This is an *.xlsm file. This file contains payment information so I can not share it. I tried to remove sheets and replace data but after edits bug is not reproducible (OpenOffice truncates blanks on save).  I can check specific sections in the document if needed.

**Comment by swmal** [comment]
Since you have a commercial license, can we move this to our online support portal? It is very difficult for us to do something about it if we cannot replicate the error. In the support portal you can upload the file privately to us and we can sign an NDA if it contains sensitive data with a commitment to destroy it after we have identified the error. If you haven't access to the support portal send an email containing your license number to info at epplussoftware dot com and you will get assistance.

**Comment by kemsky** [comment]
@swmal, I've opened ticket on the support portal.

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.5.2
